### PR TITLE
Feat authentication page

### PR DESCRIPTION
**Pull request by surajmore98**
pages added.
- register
- sign-in
- logout

Login Page

image

Logout Page

image

Register Page

image

### CODE REVIEW

File: src/cart.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+        <link rel="stylesheet" href="https://charcoal-ui.netlify.app/src/style/style.css"/>
+        <link rel="stylesheet" href="./style/nav-bar.css"/>
+        <link rel="stylesheet" href="./style/home.css"/>
+        <link rel="stylesheet" href="./style/cart.css"/>
```
Comment (shrey27) [inline src/cart.html:11]: add styles in common styles file and import the file itself

File: src/style/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    .wishlist-content {
+        grid-template-columns: repeat(2, 1fr);
+    }
+}
+
+@media only screen and (max-width: 500px) {
+    .wishlist-content {
+        grid-template-columns: 1fr;
+    }
+}
```
Comment (shrey27) [inline src/style/wishlist.css:55]: Just a suggestion. try to go with mobile first styling, that way code will be better structured, less breakpoints will be needed and you can catch the styling errors in early stages.

**Review by shrey27 — APPROVED** [review]

File: src/cart.html (hunk 1 above)
Comment (surajmore98) [reply src/cart.html:11]: Thanks for catching, will do it in next PR for all.

File: src/logout.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        <li><a class="nav-menu-link bg-white charcoal-black">Wishlist</a></li>
+    </ul>
+    <div class="main-content register-container">
+        <div class="register">
+            <div class="header">
+                <p>You are Loged Out, Please Sign in Again.</p>
+            </div>
+            <button class="btn bg-info white p-sm">Sign In</button> 
+        </div>
+    </div>
```
Comment (ddivyansh18) [inline src/logout.html]: It should say "You are Logged out"

**Review by ddivyansh18 — CHANGES_REQUESTED** [review]

File: src/logout.html (hunk 3 above)
Comment (surajmore98) [reply src/logout.html]: @ddivyansh18  done.

**Review by ddivyansh18 — APPROVED** [review]
Looks good to merge now !

**Pull request merged**